projects/storefrontlib/src/cms-components/organization/budgets/budget-form/budget-form.component.html

```diff
@@ -24,15 +24,8 @@
             required
             placeholder="{{ 'budgetForm.code.placeholder' | cxTranslate }}"
             formControlName="code"
-            [class.is-invalid]="isNotValid('code')"
           />
-          <ng-container
-            *ngIf="isNotValid('code')"
-            [ngTemplateOutlet]="validation"
-            [ngTemplateOutletContext]="{
-              control: form.controls.code
-            }"
-          ></ng-container>
+          <cx-form-errors [control]="form.get('code')"></cx-form-errors>
```

Comment:
I cannot mark this there, so I do it here. Please check line 13. This form should also be changed according to form comments I gave below.